### PR TITLE
Add org.denemo.Denemo

### PR DESCRIPTION
**Pull request by fedelibre**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. *I contributed only translations in the past. I sent a private email to Richard Shann, who gave me his Ok.*
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). *Not me, but the project owners do own the domain.*
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by fedelibre** [comment]
I'm CCing @nbenitez who helped me a lot

**Comment by fedelibre** [comment]
bot, run org.denemo.Denemo

**Comment by fedelibre** [comment]
bot, build org.denemo.Denemo

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build org.denemo.Denemo

**Comment by fedelibre** [comment]
> Build [56279 failed](https://flathub.org/builds/#/builders/32/builds/56279)

It's checking for a 128x128 icon. It cannot find it because Denemo source has only a 60px PNG icon and no SVG.
I'll ask Richard.

By the way, I've discussed with him about changing the icon installation path and the application id. He's open to accept these changes, but he's concerned about the Windows builds. I'll tackle this after getting a successful build here and replying to PR reviews.

**Comment by hfiguiere** [comment]
if you put `"skip-appstream-check": true` in a `flathub.json` it will let you have it until icon is fixed.

**Comment by barthalion** [comment]
There's also `skip-icons-check` to disable this specific test.

**Comment by hfiguiere** [comment]
I meant `skip-icon-check`, indeed.

**Comment by fedelibre** [comment]
Richard told me that the SVG was lost during the migration from cvs to git.
He pushed a 128px icon file, but there's no added value since it's simply the 60px image enlarged.
So I'll use the `skip-icons-check`.

I'm on a slow Internet connection for 10 days. When I'm back home I'll see if I can find the SVG somewhere...

**Comment by fedelibre** [comment]
bot, build org.denemo.Denemo

File: org.denemo.Denemo.yaml [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+      - type: shell
+        commands:
+          - cp -fp /usr/share/automake-*/config.{sub,guess} build-aux/
+          - cp -fp /usr/share/automake-*/config.{sub,guess} guile-readline/
+    cleanup:
+      - /include
+      - /share/aclocal
+      - /share/man
+
+  - name: pango
```
[the remark this answers is not in the document]
Comment (fedelibre) [reply org.denemo.Denemo.yaml:60]: No reason,  I just missed it was already in the SDK.

File: org.denemo.Denemo.yaml [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+
+  - name: portaudio
+    config-opts:
+      - --disable-static
+      - --without-oss
+    cleanup:
+      - /include
+    sources:
+      - type: archive
+        url: http://files.portaudio.com/archives/pa_stable_v190600_20161030.tgz
```
[the remark this answers is not in the document]
Comment (fedelibre) [reply org.denemo.Denemo.yaml:315]: Ok, thanks. Updated, I'm testing the build on my laptop

**Comment by fedelibre** [comment]
bot, build org.denemo.Denemo

**Comment by barthalion** [comment]
Does Richard Shann or someone else from the team has GitHub account so we can invite them?

**Comment by fedelibre** [comment]
No, Richard doesn't have a Github account and he already told me he's not interested in maintaining the flatpak repository.

I think Jeremiah may be interested, so I think you can add him.
@jjbenham do you want to be invited to have write access to the Denemo flatpak repository?

**Comment by jjbenham** [comment]
I will take care of that soon. I think I have to user accounts jjbenham and
jbenham. I am going to try to switch to jbenham if I can recover my
credentials.

Best,
Jeremiah

**Comment by nedrichards** [comment]
I can see various bits of icon chat but fwiw the icon in the about view is missing. It's not a blocker and I'll merge shortly but just to let you know.

**Comment by fedelibre** [comment]
> I can see various bits of icon chat but fwiw the icon in the about view is missing. It's not a blocker and I'll merge shortly but just to let you know.

This should be fixed when upgrading to the next release of Denemo, as in the master branch the application id is now following the reverse-DNS schema.

**Pull request closed without merging**